server/asset/btc/btc_test.go

```diff
@@ -301,6 +301,17 @@ func txOutID(txHash *chainhash.Hash, index uint32) string {
 	return txHash.String() + ":" + strconv.Itoa(int(index))
 }
 
+const optimalFeeRate uint64 = 24
+
+func (testNode) EstimateSmartFee(confTarget int64, mode *btcjson.EstimateSmartFeeMode) (*btcjson.EstimateSmartFeeResult, error) {
+	optimalRate := float64(optimalFeeRate) * 1e-5
+	// fmt.Println((float64(optimalFeeRate)*1e-5)-0.00024)
```

Comment:
BTW, the kind of thing that makes be super uncomfortable with floating point math: https://play.golang.org/p/Dhmxmp5OosM

The error is less than 1e-8, so it's going to work out to the same atoms/byte, but it still sucks.